From 4f69ffc20e9ce1d9d78c466e73104f3fd5a034df Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <xiaowen.liu@freescale.com>
Date: Mon, 31 Aug 2015 16:28:18 +0800
Subject: [PATCH 531/602] MA-6469 fix screen flash before system enter
 suspend.

Wallpaper display before screen off animation introduce this issue.
Set config_dozeAfterScreenOff to true to advance screen off animation.

Signed-off-by: Liu Xiaowen <xiaowen.liu@freescale.com>
---
 evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml       | 4 ++++
 sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml  | 4 ++++
 sabreauto_6sx/overlay/frameworks/base/core/res/res/values/config.xml | 4 ++++
 sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml   | 4 ++++
 sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml   | 4 ++++
 sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml    | 4 ++++
 6 files changed, 24 insertions(+)

diff --git a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
index 226b65d..ac67141 100644
--- a/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/evk_6sl/overlay/frameworks/base/core/res/res/values/config.xml
@@ -203,4 +203,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
index 4d49876..c4dde0f 100755
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
@@ -223,4 +223,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
diff --git a/sabreauto_6sx/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6sx/overlay/frameworks/base/core/res/res/values/config.xml
index 70607f3..ada401b 100755
--- a/sabreauto_6sx/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6sx/overlay/frameworks/base/core/res/res/values/config.xml
@@ -223,4 +223,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
diff --git a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
index a19ef85..d2495d3 100755
--- a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/config.xml
@@ -222,4 +222,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
diff --git a/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
index 5edb7b0..ec305d7 100755
--- a/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_6sx/overlay/frameworks/base/core/res/res/values/config.xml
@@ -214,4 +214,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
diff --git a/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
index 6721aa8..34f7d63 100644
--- a/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
@@ -211,4 +211,8 @@
     -->
     <bool name="config_ui_enableFadingMarquee">false</bool>
 
+    <!-- If true, the doze component is not started until after the screen has been
+         turned off and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOff">true</bool>
+
 </resources>
-- 
1.8.0

